pkg/system/filesys_windows.go

```diff
@@ -51,7 +50,7 @@ func mkdirall(path string, applyACL bool, sddl string) error {
 		return &os.PathError{
 			Op:   "mkdir",
 			Path: path,
-			Err:  syscall.ENOTDIR,
+			Err:  windows.ERROR_PATH_NOT_FOUND,
```

Comment:
Pretty sure this is OK, but can you confirm no higher level caller in the codebase takes a dependency on a specific error. Also applies in `OpenFileSequential`

Reply:
Hm, good point; given that we documented `MkdirAll` above as a "drop in" replacement for `os.MkdirAll`, perhaps it's too risky. I couldn't find an issue at a glance, but given that it's only a small enhancement, let's revert these